src/core/Akka.Tests/Configuration/ConfigurationSpec.cs

```diff
@@ -63,7 +64,6 @@ public void The_default_configuration_file_contain_all_configuration_properties(
             settings.SchedulerClass.ShouldBe(typeof (HashedWheelTimerScheduler).FullName);
         }
 
-#if CONFIGURATION
```

Comment:
This is still required here, since `ConfigurationManager` does not seem to load `App.config` file under .net core. And `CONFIGURATION` is defined only for full .net framework.

Reply:
@IgorFedchenko thanks Igor - I'll look at that. I wasn't getting that issue locally but the test runner will tell me for sure. Just updated to HOCON v1.3.1 to pare down the number of HOCON-related bugs in this PR, although I know more are hiding around the corner.